packages/core/js-client/src/connection/RelayConnection.ts

```diff
@@ -180,19 +180,27 @@ export class RelayConnection implements IConnection {
       );
     }
 
-    log.trace("sending particle...");
-
     // Reusing active connection here
     const stream = await this.lib2p2Peer.dialProtocol(
       this.relayAddress,
       PROTOCOL_NAME,
     );
 
-    log.trace("created stream with id ", stream.id);
+    log.trace(
+      "sending the particle %s to the created stream %s",
+      particle.id,
+      stream.id,
```

Comment:
Maybe better print relayAddress here? `stream.id` is a very low-level thing, afaik.